scrapy/czlscrape/spiders/comunicatii.py

```diff
@@ -0,0 +1,119 @@
+import hashlib
+from datetime import *
```

Comment:
PEP8 (the Python style guide) [zice](https://www.python.org/dev/peps/pep-0008/#imports) să evităm `import *` pentru că nu mai e clar de unde e importat fiecare simbol.